rocAL/include/meta_data/meta_data_reader.h

```diff
@@ -100,4 +100,5 @@ class MetaDataReader {
     virtual ImgSize lookup_image_size(const std::string& image_name) { return {}; }
     virtual void set_aspect_ratio_grouping(bool aspect_ratio_grouping) { return; }
     virtual bool get_aspect_ratio_grouping() const { return {}; }
+    virtual std::vector<std::string> get_relative_file_path() { return {}; } // Returns the relative file_path's of the reader 
```

Comment:
why is this related to last_batch policy? Can't you derive the relative path instead?

Reply:
This is not related to last batch policy - this is used in source evaluation